lib/rules/require-capitalized-comments.js

```diff
@@ -66,25 +137,44 @@ module.exports.prototype = {
     check: function(file, errors) {
         var inTextBlock = null;
 
+        var exceptions = this._exceptions || {};
+        var hasExceptions = this._hasExceptions;
+
         var letterPattern = require('../../patterns/L');
         var upperCasePattern = require('../../patterns/Lu');
 
         file.getComments().forEach(function(comment) {
-            var stripped = comment.value.replace(/[\n\s\*]/g, '');
```

Comment:
I thought it would save having to do unnecessary text replacement, and related operations, if it is an exception, which now would cause an early return.